src/layouts/PageContainer.tsx

```diff
@@ -2,16 +2,18 @@ import { useFlags } from "launchdarkly-react-client-sdk";
 import useMediaQuery from "@mui/material/useMediaQuery";
 import theme from "styles/theme";
 import Container from "@mui/material/Container";
-import React from "react";
+import React, { CSSProperties } from "react";
 
 export const DesktopContainer = ({ children, style }: { children; style? }) => {
   return (
     <div
       style={{
         backgroundColor: "primary.main",
         maxWidth: 1096,
-        margin: "2rem",
-        paddingBottom: "3rem",
+        marginLeft: "2rem",
+        marginRight: "2rem",
+        marginTop: "2rem",
+        marginBottom: "5rem",
```

Comment:
You can also do https://www.w3schools.com/css/css_margin.asp
```suggestion
        margin: "2rem 2rem 5rem",
```

Reply:
for some reason this wouldn't actually work for me on the X axis... I am not sure why. also tried marginX and nada. I will try again though maybe I did something weird.